inst/schema/ValidateInputResponse.schema.json

```diff
@@ -25,9 +25,13 @@
         "data": { "$ref": "ShapeResponseData.schema.json" },
         "filters": {
           "type": "object",
-          "properties" :{
-            "regions": {
+          "properties" : {
+            "level_labels": {
               "type": "array",
+              "items": { "$ref": "LevelLabels.schema.json"}
+            },
+            "regions": {
+              "type": "object",
               "items": { "$ref": "NestedFilterOption.schema.json" }
```

Comment:
It looks like this schema does not match the data:

```
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "type": "object",
  "properties": {
    "name": {
      "type": "string"
    },
    "id": {
      "type": "string"
    },
    "options": {
      "type": "array",
      "items": {
        "type": "object"
      }
    }
  },
  "additionalProperties": false,
  "required": [
    "name", "id"
  ]
}
```

it should be recursive, right?

Reply:
Fixed that.